### PR TITLE
V3/optimisation

### PR DESCRIPTION
**Pull request by AntonNazarenko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -161,6 +152,7 @@
     "dotenv-expand": "4.2.0",
     "email-validator": "^2.0.4",
     "eslint": "^5.16.0",
+    "eslint-config-airbnb": "^17.1.0",
```
Comment (JacobPlaster) [inline package.json]: should be in dev deps

Comment (JacobPlaster) [reply package.json]: also the 'eslint' on the line above

File: package.json [hunk 2]
```diff
@@ -182,6 +174,8 @@
     "js-levenshtein": "^1.1.6",
     "lodash": "^4.17.11",
     "mini-css-extract-plugin": "0.5.0",
+    "node-sass-chokidar": "^1.3.5",
```
Comment (JacobPlaster) [inline package.json]: Do we need this? If so, can we move to dev-deps as well?

File: package.json (hunk 1 above)
Comment (AntonNazarenko) [reply package.json]: the app doesn't work without this dependencies,  so i moved it to non-dev dependencies
the same problem with dependencies below
already removed all unused dependencies

**Pull request merged**